docs/asb/README.md

```diff
@@ -0,0 +1,134 @@
+# XMR to BTC Atomic Swap - Automated Swap Backend (ASB)
+
+## Quick Start ASB
+
+- Ensure that you have the Monero Wallet RPC running with `wallet-dir` and `disable-rpc-login`:
+  - `monero-wallet-rpc --stagenet --daemon-host STAGENET-NODE-URL --rpc-bind-port STAGENET-NODE-PORT --disable-rpc-login --wallet-dir PATH/TO/WALLET/DIR`
+- Run the ASB in terminal: `./asb start`
+- Follow the setup wizard in the terminal
+
+Public stagenet nodes:
+
+- `monero-stagenet.exan.tech`, `38081`
+- `stagenet.community.xmr.to`, `38081`
+
+Run `./asb --help` for more information.
+
+## ASB Details
+
+The ASB is a long running daemon that acts as the trading partner to the swap CLI.
+The CLI user is buying XMR (i.e. receives XMR, sends BTC), the ASB service provider is selling XMR (i.e. sends XMR, receives BTC).
+The ASB can handle multiple swaps with different peers concurrently.
+
+The ASB reacts to:
+
+- Quote requests from a CLI
+- Swap execution setup with a CLI
+- Swap execution messages with a CLI
+
+Both the ASB and the CLI can be run by anybody.
+The CLI is designed to run one specific swap against an ASB.
+The ASB is designed to run 24/7 as a daemon that responds to CLIs connecting.
+Since the ASB is a long running task we specify the person running an ASB as service provider.
+
+### ASB discovery
+
+Currently, there is no automated discovery for service providers running an ASB.
+A service provider has to manually provide the connection details to users that will run the CLI.
+
+Currently the connection details consist of:
+
+- Multiaddress
+- Peer-ID
+
+The Peer-ID is printed upon startup of the ASB.
+The multi-address is the IP-address or URL (if DNS entry configured) of the service provider.
```

Comment:
That is technically not fully correct. See [here](https://docs.libp2p.io/concepts/addressing/) for the official documentation. Might also be worthwhile linking to.